src/main/resources/db/migration/0250/V276__NewParticipantProjectSpeciesFields.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE accelerator.participant_project_species ADD COLUMN created_by BIGINT REFERENCES users (id);
+ALTER TABLE accelerator.participant_project_species ADD COLUMN created_time TIMESTAMP WITH TIME ZONE NOT NULL;
```

Comment:
This migration will fail if there are any existing rows in the table (which there are on staging; I just checked.)

Adding a non-nullable column needs three steps:
```sql
ALTER TABLE foo ADD COLUMN bar TIMESTAMP WITH TIME ZONE;
UPDATE foo SET bar = NOW();
ALTER TABLE foo ALTER COLUMN bar SET NOT NULL;
```

Reply:
Thanks for that, I see plenty of examples for the same treatment for user ID too.